docs: :pencil2: small edits to architecture and naming page (#922)
seedcase-project/seedcase-sprout: docs/design/index.qmd

---
title: "Design"
---

The core aim of Sprout is to **take data and metadata and convert them
into a standardized and organized storage structure** that follows best

> practices for data engineering, at least within a research context.

Reviewer: ```suggestion
practices for data engineering, particularly within a research context.
```
or 
```suggestion
practices for data engineering, particularly with a focus on research contexts.
```

Specifically, Sprout aims to:

> 1.  Take generated data from various source locations (such as the
>     clinic or laboratories), that may be distributed geographically or
>     organizationally, and store it in a standardized and efficient
>     format.

Reviewer: ```suggestion
1.  Take generated data from various source locations (such as
    clinics or laboratories), which may be distributed geographically or
    organizationally, and store it in a standardized and efficient
    format.
```

> 2.  Require metadata be included for the data and organized following a

Reviewer 2: ```suggestion
2.  Ensure that metadata is included for the data and organized following a
```

> 2.  Ensure that metadata is included for the data and organized following a
>     standardized format, in a way that explicitly and programmatically
>     links the metadata to the data.

Reviewer: ```suggestion
2.  Ensure that metadata is included for the data and organized in a
    standardized format, explicitly and programmatically
    linking the metadata to the data.
```

> The purpose of these documents are to describe the overall design of

Reviewer 2: ```suggestion
The purpose of these documents is to describe the overall design of
```

> Sprout in enough detail to primarily help us develop Sprout in a way

Reviewer 2: ```suggestion
Sprout in enough detail to help us develop Sprout in a way
```

> The purpose of these documents is to describe the overall design of
> Sprout in enough detail to help us develop Sprout in a way
> that is maintainable over the long term and that ensures we as a team
> are on the same page on what Sprout is and is not.

Reviewer: ```suggestion
The purpose of these documents is to describe the overall design of
Sprout in enough detail to help us develop it in a way
that is sustainable (i.e., maintainable over the long term) and that ensures we as a team
have a shared understanding of what Sprout is and is not.
```
